modules/core/include/opencv2/core/cuda.hpp

```diff
@@ -569,18 +569,18 @@ class CV_EXPORTS HostMem
     hardware. Laptops often share video and CPU memory, so address spaces can be mapped, which
     eliminates an extra copy.
      */
-    GpuMat createGpuMatHeader() const;
+    CV_WRAP GpuMat createGpuMatHeader() const;
```

Comment:
Perhaps it not usable in bindings due missing access to memory pointer.

Reply:
Wrapper removed